routes/index.js

```diff
@@ -53,4 +53,8 @@ router.get('/', function(req, res) {
   });
 });
 
+router.get('/about', function(req, res) {
```

Comment:
can u move this whole route into app.js so that the user does not have to be logged in to read the about page?

Reply:
it would be `app.get` instead of `router.get`